zbus_macros/tests/tests.rs

```diff
@@ -18,6 +18,30 @@ mod param {
     }
 }
 
+// Test that the `object` attribute works on properties (issue #356)
```

Comment:
Kudos for adding a test. 👍 However it doesn't actually test the functionality. For proper-test, you'll want to use the `interface` macro here to generate both the interface and proxy, start the interface and retrieve the property.

There are some p2p tests in `zbus::connection` module you can copy the boilerplate from.